task/nucleo-user-button/src/main.rs

```diff
@@ -67,10 +67,26 @@ pub fn main() -> ! {
     let user_leds = drv_user_leds_api::UserLeds::from(USER_LEDS.get_task_id());
     let sys = drv_stm32xx_sys_api::Sys::from(SYS.get_task_id());
 
-    let (led, rising, falling) = if let Some(config) = TASK_CONFIG {
-        (config.led, config.rising, config.falling)
+    let (led, sensitivity) = if let Some(Config {
+        rising,
```

Comment:
fwiw, if you were to `derive(Deserialize)` on `Edge` you could use the strings "Rising", "Falling", "Both" in the config.

Reply:
oh, good call. that would make the code here much simpler!